Feature zoom image

shruthihsrinath opened the pull request.

shruthihsrinath committed:
add zoom image feature for portfolio images

shruthihsrinath committed:
Adjust website features for resizing browser window

shreeshbhat commented on src/app/commons/header/header.component.ts:
```diff
@@ -6,5 +6,13 @@ import { Component } from '@angular/core';
+  }
 
+  onVisited() {
+    this.isToggle = !this.isToggle;
```
Both the functions are doing the same logic. You can collapse these down to one function.

shreeshbhat commented on src/app/portfolio/portfolio.component.ts:
```diff
@@ -21,15 +28,70 @@ export class PortfolioComponent implements OnInit {
+      this.portfolioImages = data.filter(collab => collab.collabName == 'PORTFOLIO');
+    });
+
+    if (window.innerWidth <= 810) {
```
You can collapse this if check to a condition assignment.

shreeshbhat commented on src/app/portfolio/portfolio.component.ts:
```diff
@@ -21,15 +28,70 @@ export class PortfolioComponent implements OnInit {
+        this.isMobile = true;
+      }
+      else {
+        this.isMobile = false;
```
Similar comment here about collapsing single line if check

shruthihsrinath committed:
Resolved comments on feature-zoom-image
Resolve functionality repetition for toggleMenu

Collapse if check to a conditional assigment

shruthihsrinath committed:
Style changes for different media

shruthihsrinath committed:
Style change: Hamburger menu easy transition

shruthihsrinath committed:
Style change: Remove overscroll behavior

shruthihsrinath committed:
Remove unwanted comments
